types/ember-power-select/components/power-select-multiple/trigger.d.ts

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+
+import { Select } from '../power-select';
+
+interface PowerSelectMultipleTriggerArgs<O> {
+    select: Select;
+    placeholder?: string;
+    searchField: string;
+    onInput?: (e: InputEvent) => boolean;
+    onKeydown?: (e: KeyboardEvent) => boolean;
+    buildSelection: (lastSelection: O, select: Select) => any[];
```

Comment:
Is it possible that `lastSection` could be `null` here?

Reply:
I believe this came right from their generated TS file. So not according to them?